implementation/jwt-auth/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -217,6 +218,13 @@ private static JWTAuthContextInfoProvider create(String publicKey,
     @Inject
     @ConfigProperty(name = "mp.jwt.verify.publickey", defaultValue = NONE)
     private String mpJwtPublicKey;
+
+    /**
+     * @since 1.1
+     */
+    @Inject
+    @ConfigProperty(name = "smallrye.jwt.verify.secretkey", defaultValue = NONE)
+    private String mpJwtSecretKey;
```

Comment:
Suggested adjustment to JavaDoc and field name (non-MP property)
```suggestion
    /**
     * @since 4.5.4
     */
    @Inject
    @ConfigProperty(name = "smallrye.jwt.verify.secretkey", defaultValue = NONE)
    private String jwtSecretKey;
```